tagging.md

```diff
@@ -95,5 +95,5 @@ means it contains 10 tracks. If it's MOVEMENT (20), that means it's 10 movements
 The following is a complete list of the supported Matroska Tags. While it is possible
 to use Tag names that are not listed below, this is not recommended as compatibility will
 be compromised. If you find that there is a Tag missing that you would like to use,
-then please contact the Matroska team for its inclusion in the specifications before the format reaches 1.0.
+then please contact the IANA for its inclusion in the Mastroska Tags Registry, see (#matroska-tags-names-registry).
```

Comment:
`contact the IANA` or `contact IANA`?

Reply:
I mofified the text slightly as IANA is not a person to contact. The registry actually contains the names of people to contact.